src/test/java/uk/gov/hmcts/reform/sscs/service/SubmitAppealServiceTest.java

```diff
@@ -108,10 +130,19 @@ public void setUp() {
 
         deserializer = new SubmitYourAppealToCcdCaseDataDeserializer();
 
+        ResponseEntity<Resource> mockResponseEntity = mock(ResponseEntity.class);
+        ByteArrayResource stubbedResource = new ByteArrayResource(new byte[] {});
+        when(mockResponseEntity.getBody()).thenReturn(stubbedResource);
+
+        when(authTokenGenerator.generate()).thenReturn("token");
+        when(documentDownloadClientApi.downloadBinary(anyString(), anyString(), anyString())).thenReturn(mockResponseEntity);
+
+        evidenceManagementService = new EvidenceManagementService(authTokenGenerator, documentUploadClientApi, documentDownloadClientApi);
```

Comment:
@oleg-hmcts , the code looks good. My only concern really is that this is a Junit test and you are using it to cover the EvidenceManagementService so it looks like that should be moved to an integration test really. What do you think? I am happy to leave it as it is for now, though. But wanted to get your feedback.

Reply:
Actually, discard the above comment, please. I just noticed that other junit tests are using the same approach. Let's leave it as it is for now. Thanks.